omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
-gem "omnibus", git: "https://github.com/chef/omnibus.git", branch: "master"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git", branch: "master"
+gem "omnibus", git: "https://github.com/chef/omnibus.git", branch: "jm/deep_sign"
```

Comment:
Reminder to update these to master before merging

Reply:
We are going to merge pointed at these branches to test get a package out and flip to master once the underlying branches merge